layers/README.md

```diff
@@ -119,3 +129,52 @@ lo4 = LayerOps(score=lambda x: '; '.join(x),
 out_layer6 = lo4.process([demo2, demo3])                      # Trigger processing on a list of demo2 and demo0
 out_layer6.to_file("C:\demo_layer6.json")                     # Save combined comment layer to file
 ```
+
+## to_excel.py
+to_excel.py provides the ToExcel class, which is a way to export an existing layer file as an Excel 
+spreadsheet. The ToExcel class has an optional parameter for the initialization function, that 
+tells the exporter to build the output matrix based on live data from cti-taxii.mitre.org. It 
+can also use cached subtechniques data. If neither of these is used, it will default to pulling from the 
+ATT&CK cti repo. If matrix template lacks elements included in a layer's technique listing, those 
+elements will not be visible in the output file.
```

Comment:
> If matrix template lacks elements included in a layer's technique listing, those 
elements will not be visible in the output file.

This is a bit unclear. Probably rewrite as:

> If the layer defines annotations on techniques which are not present in the matrix data, they will not be included in the output file

However, I'm not sure we even need to say this as it's sorta part of the Navigator design already and you'd think people would realize that we can't annotate things which don't exist.

Reply:
I can pull it out. I originally wrote that because I wanted some context for the warning message the code throws when that situation occurs, but given that it's part of the Navigator design, I think it's reasonable that it would follow from that.